en/lessons/computer-vision-deep-learning-pt1.md

```diff
@@ -0,0 +1,580 @@
+---
+title: "Computer Vision for the Humanities: An Introduction to Deep Learning for Image Classification (Part 1)"
+collection: lessons
+layout: lesson
+slug: computer-vision-deep-learning-pt1
+date: 2021-01-26
```

Comment:
You'll want to update the publication date for both lessons (and remove the translation date)

Reply:
thanks! i made those updates. just not sure why the build broke